_build/pipelines/release.yml

```diff
@@ -0,0 +1,191 @@
+name: $(date:yyyyMMdd)$(rev:.r)
+
+trigger: none
+
+pr: none
+
+resources:
+  repositories:
+  - repository: self
+    type: git
+    ref: refs/heads/main
+
+jobs:
+- job: Build
+  displayName: Build
+  timeoutInMinutes: 240
+
+  strategy:
+    matrix:
+      WinUICs:
+        Moniker: WinUICs
+        BuildConfiguration: ReleaseWinUICs
+        BuildPlatform: Any CPU
+      WinUICpp:
+        Moniker: WinUICpp
+        BuildConfiguration: ReleaseWinUICpp
+        BuildPlatform: Any CPU
+      WPF:
+        Moniker: WPF
+        BuildConfiguration: ReleaseWPF
+        BuildPlatform: Any CPU
+      UWP:
+        Moniker: UWP
+        BuildConfiguration: ReleaseUWP
+        BuildPlatform: Any CPU
+
+  pool:
+    vmImage: windows-2022
+
+  steps:
+  - checkout: self
+    clean: true
+    submodules: recursive
```

Comment:
shouldn't need any submodules - the one we did use (CoreTS) isn't used any more.